proxy/describe.go

```diff
@@ -8,30 +8,27 @@ import (
 	"fmt"
 	"io/ioutil"
 	"net/http"
-	"net/url"
-	"path"
 
 	types "github.com/openfaas/faas-provider/types"
 )
 
 //GetFunctionInfo get an OpenFaaS function information
 func GetFunctionInfo(gateway string, functionName string, tlsInsecure bool) (types.FunctionStatus, error) {
-	return GetFunctionInfoToken(gateway, functionName, tlsInsecure, "")
+	return GetFunctionInfoToken(gateway, functionName, tlsInsecure, "", "")
```

Comment:
Same comment as https://github.com/openfaas/faas-cli/pull/694/files#r328597947

Reply:
All functions without a `Token` suffix need to pass through all the options. The empty string is here because the first method doesn't accept a token and the second does. The namespace must pass-through just like the `functionName`.

Reply:
Done :+1: